tasks/tinc_configure.yml

```diff
@@ -103,3 +125,25 @@
   systemd:
     name: "{{ tinc_service_name }}"
     enabled: yes
+  when: ansible_service_mgr == "systemd"
+
+# this enables the use of the tinc start script provided by OpenWrt
+- name: OpenWRT Configure tinc with UCI
```

Comment:
I am sorry to ask more things, could we move that to main, next to the systemd start script templating? It's easier to read that way.

Reply:
agreed.